docs/nextflow.md

```diff
@@ -0,0 +1,6 @@
+# Casanovo Nextflow Workflow
+
+To simplify the process of setting up and running Casanovo, a dedicated [Nextflow](https://www.nextflow.io/) workflow is available.
+In addition to automatic the installation of Casanovo and its dependencies, the Casanovo Nextflow workflow provides an automated mass spectrometry data pipeline that converts input data files to a Casanovo compatible format using [msconvert](https://bioinformaticshome.com/tools/proteomics/descriptions/msconvert.html#gsc.tab=0), infers peptide sequences using Casanovo, and (optionally) uploads the results to [Limelight](https://limelight-ms.org/) - a platform for sharing and visualizing proteomics results.
```

Comment:
**typo:** "automatic the installation" -> "automate" I think? Although it's twice "automate" in close succession, so small rephrasing recommended. The first one "simplify"?